packages/web/src/components/ui/Package/Package.tsx

```diff
@@ -19,19 +27,82 @@ export type Props = {
 };
 
 export default function Package({ className, variant = 'grid', pkg }: Props) {
+  const repositoryUrl = pkg.registryMetadata?.repositoryUrl;
+  const homepageUrl = pkg.registryMetadata?.homepageUrl;
+  const isOutdated =
+    pkg.registryMetadata && semver.gtr(pkg.registryMetadata.latestVersion, pkg.packageVersionRange);
 
   return (
     <div className={clsx(styles.container, styles[variant], className)}>
-      <div className={styles.main}>
-        <div className={styles.name}>{pkg.packageName}</div>
-        <div className={styles.version}>{pkg.packageVersionRange}</div>
+      <div className={styles.registryMeta}>
+        {isOutdated && <div className={styles.badge}>Outdated</div>}
+        <div className={styles.externalLinks}>
+          {repositoryUrl && (
+            <a
+              href={repositoryUrl}
+              target='_blank'
+              rel='noopener noreferrer'
+              className={styles.externalLink}
+            >
+              <Github />
+            </a>
+          )}{' '}
```

Comment:
This space will appear always, which is probably not the desired behavior. This should be done with css, I guess.

Reply:
Fixed